### DISCUSSION TITLE
Bring on more contributors and maintainers

### DISCUSSION THREAD

**Discussion by sandstrom** [jdx/mise#3487] [General]
Mise is great, played around with it a bit recently.

1. However, this is a type of project that (when popular) will have a lot of feature requests, but reports etc. Maybe you should consider bringing on board 1-2 team members to help out. 

2. Another thing that might be useful is to define some clear boundaries or anti-goals (it's okay if these change sometimes), or a list of supported platforms, so that it's easier to dismiss things that are 'out of scope'. As a security precaution (after XZ/JiaT75), it might make sense to ask someone (if the initiative is from you, it's less risky).

3. A third suggestion is to do frequent issue gardening, perhaps using a bot, and basically just shut down issues that have little activity or no associated PR. Basically, if someone reports a bug that affect few people, and cannot fix it themselves, and no one else does within (say) 3 months, it's probably better to just close it.
    
    Two approaches I've seen:
    
    - https://github.com/renovatebot/renovate
    - https://github.com/probot/probot

**Comment by eisraeli** [comment]
1. This should have gone under 'discussions' rather than 'issues'.
2. This project is defiantly at risk by the [bus factor](https://en.wikipedia.org/wiki/Bus_factor). 
3. I think mise is a good step forward from ASDF, but I still feel it lacks security, a framework and much more community involvement due to lack of reliable maintainers.

**Comment by jadejr** [comment]
> However, this is a type of project that (when popular) will have a lot of feature requests, but reports etc. Maybe you should consider bringing on board 1-2 team members to help out.

And you can be one of them.  There are no maintainers to bring on except for folks like you

**Comment by sandstrom** [comment]
@jadejr I know, and I do maintain some smaller open-source projects. 

That said, I don't know Rust very well, so I don't think this is where I'd be the most helpful.

**Comment by ssbarnea** [comment]
I am really glad others raised this. I learnt the hard way that projects that survive the test of time are those that have multiple maintainers.
